vendor/k8s.io/kubernetes/pkg/client/restclient/request.go

```diff
@@ -888,16 +892,49 @@ func (r *Request) transformResponse(resp *http.Response, req *http.Request) Resu
 			body = data
 		}
 	}
-	glog.V(8).Infof("Response Body: %s", string(body))
+
+	if glog.V(8) {
+		switch {
+		case bytes.IndexFunc(body, func(r rune) bool { return r < 0x0a }) != -1:
```

Comment:
It's looking for control characters that are unprintable in strings - i'll
add a comment to the upstream PR.

On Fri, Jul 22, 2016 at 1:11 PM, David Eads <notifications@github.com>
wrote:

> In vendor/k8s.io/kubernetes/pkg/client/restclient/request.go
> <https://github.com/openshift/origin/pull/9814#discussion_r71912474>:
>
> > @@ -888,16 +892,49 @@ func (r *Request) transformResponse(resp *http.Response, req *http.Request) Resu
> >  			body = data
> >  		}
> >  	}
> > -	glog.V(8).Infof("Response Body: %s", string(body))
> > +
> > +	if glog.V(8) {
> > +		switch {
> > +		case bytes.IndexFunc(body, func(r rune) bool { return r < 0x0a }) != -1:
>
> deserves a comment. I'm assuming this is a marker for protobuf?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9814/files/cc7c3b48eb97a0f92e55097d86df008c853311ac#r71912474>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_p3zDGw2iLNBKC0pyk8jHIYHHRlgjks5qYPnLgaJpZM4JLFc7>
> .
>